docs/docs/02-configuration/11-runtime-target-type-mapping.md

```diff
@@ -0,0 +1,39 @@
+# Runtime target type mapping
+
+If the target type of a mapping is not known at compile time,
+a mapping method with a `Type` parameter can be used.
+Mapperly implements this mapping method
+using all mappings the user defined in the mapper.
+
+```csharp
+[Mapper]
+public static partial class ModelMapper
+{
+    // highlight-start
+    public static partial object Map(object source, Type targetType);
+    // highlight-end
+
+    private static partial AudiDto MapAudi(Audi source);
+    private static partial PorscheDto MapPorsche(Porsche source);
+}
+
+class Audi {}
```

Comment:
IMO we should not use commercial brands in our documentation.

Reply:
Used fruits instead.